content/issues/2025-08-10-onlyoffice.md

```diff
@@ -0,0 +1,12 @@
+---
+title: 'New Container: ONLYOFFICE'
+date: '2025-08-10 23:25:00Z'
+informational: true
+affected:
+  - 'New Containers'
+  - 'ONLYOFFICE'
```

Comment:
Can you do the tag as all lowercase, makes it easier keeping track when they're all consistent.